File: A/c_plus_equal.py
import sys
input = sys.stdin.readline

def solve():
    a, b, n = map(int, input().split())
    ops = 0
    while a <= n and b <= n:
        a, b = (a, b + a) if a >= b else (a + b, b)
        ops += 1
    print(ops)

for _ in range(int(input())):
    solve()

File: A/c_plus_equal.c
#include <stdio.h>

void solve() {
    long long a, b, n;
    scanf("%lld %lld %lld", &a, &b, &n);
    int ops = 0;
    while (a <= n && b <= n) {
        if (a >= b) {
            b += a;
        } else {
            a += b;
        }
        ++ops;
    }
    printf("%d\n", ops);
}

int main(void) {
    int t;
    scanf("%d", &t);
    while (t--) {
        solve();
    }
    return 0;
}
